src/FloatingButton/FloatingButton.less

```diff
@@ -0,0 +1,81 @@
+@import (reference) "../less/utilities";
+
+.FloatingButton--container {
+  position: fixed;
+  .zIndex--9();
+  pointer-events: none;
+}
+
+.FloatingButton--left {
+  .items--start();
+}
+
+.FloatingButton--center {
+  .items--center();
+  left: 50%;
+  transform: translate(-50%, 0);
+}
+
+.FloatingButton--right {
+  .items--end();
+}
+
+.FloatingButton--bottom {
+  .flex--direction--columnReverse();
+}
+
+.FloatingButton--top {
+  .flex--direction--column();
+}
+
+.FloatingButton--subElement {
+  .margin--2xs();
+  pointer-events: auto;
+}
+
+.FloatingButton--animate {
+  transition: opacity @timingPromptly @motionSmooth, transform @timingPromptly @motionSmooth;
+}
+
+button.FloatingButton--button {
+  .boxShadow--heavy();
+  border-radius: 500rem;
+}
+
+.FloatingButton--inactive--bottom {
+  transform: translateY(2.5rem);
```

Comment:
To the best of my understanding, this style is only necessary for animation. It took me a while to process that. (I had to run the code locally to fully make that connection :smile:) I might leave a comment noting that